tests/testutil/service.go

```diff
@@ -121,7 +122,13 @@ func (ts *TestService) GetEndpoint() string {
 	if err != nil {
 		ts.T.Fatal(err.Error())
 	}
-	response, err := WaitForURLResponse(endpoint)
+	return ts.GetEndpointResponse(endpoint)
+}
+
+// GetEndpointResponse performs an http.get against the service endpoint and returns response if
```

Comment:
Can you move all the "getEndpoint..." methods to the getters section below this so they are all in same location.